matter_server/server/device_controller.py

```diff
@@ -1050,48 +1050,30 @@ async def _call_sdk(self, func: Callable[..., _T], *args: Any, **kwargs: Any) ->
             ),
         )
 
-    async def _check_interview_and_subscription(
-        self, node_id: int, reschedule_interval: int = 30
-    ) -> None:
-        """Handle interview (if needed) and subscription for known node."""
-
+    async def _setup_node(self, node_id: int) -> None:
+        """Handle set-up of subscriptions and interview (if needed) for known/discovered node."""
         if node_id not in self._nodes:
             raise NodeNotExists(f"Node {node_id} does not exist.")
 
         # (re)interview node (only) if needed
-        node_data = self._nodes.get(node_id)
+        node_data = self._nodes[node_id]
         if (
-            node_data is None
+            # re-interview if we dont have any node attributes (empty node)
+            not node_data.attributes
             # re-interview if the schema has changed
-            or node_data.interview_version < SCHEMA_VERSION
+            or node_data.interview_version < DATA_MODEL_SCHEMA_VERSION
```

Comment:
People which installed the beta have now interview version set to 7. This means next time we bump `DATA_MODEL_SCHEMA_VERSION` they won't reinterview, which could be problematic :grimacing: 

Two ways: Next time we need to jump to 8, or we use != here... I think I prefer != here, so we can also downgrade. Thoughts?

Reply:
lets go for the non equals

Reply:
adjusted